libs/application/ui-fields/README.md

```diff
@@ -9,3 +9,112 @@ Each Field has to have its own unique `type` which needs to be part of the `Fiel
 ## Running unit tests
 
 Run `nx test application-ui-fields` to execute the unit tests via [Jest](https://jestjs.io).
+
+---
+
+## FileUploadFormField
+
+The file upload form field provides a general way to upload files for your application.
+
+### To use the FileUploadFormField in an application
+
+1. Add it to the _schema_ using the following structure:
+
+```
```

Comment:
You can add `typescript` to get nice color syntax for the following code snippets

```suggestion
```typescript
```

Reply:
Oh nice thanks! Updated now.